tests/cli/test_cmds_mark_for_deployment.py

```diff
@@ -424,6 +424,7 @@ def test_MarkForDeployProcess_get_authors_diffs_against_prod_deploy_group(
         auto_rollback_delay=1,
         authors=["fakeuser1"],
     )
+    mfdp.get_authors()
```

Comment:
is this just to make the assertion on the next line pass? If so, I'd just delete that assertion.